Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+            "helpMarkDown": "Time (provide unit of time (s, m, h)) to wait before commencing collection and analysis of metrics"
+        },
+        {
+            "name": "lifetime",
+            "type": "string",
+            "label": "Lifetime",
+            "defaultValue": "10m",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == realtime",
+            "helpMarkDown": "Time (provide unit of time (s, m, h))) spanning the entire duration of the canary analysis"
+        },
+        {
+            "name": "analysisBeginTime",
+            "type": "string",
+            "label": "Analysis begin time",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == range",
+            "helpMarkDown": "Start time to collect the metrics. Provide timestamp in yyyy-mm-ddThh:mm:ssZ format (UTC time)"
+        },
+        {
+            "name": "analysisEndTime",
+            "type": "string",
+            "label": "Analysis end time",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == range",
+            "helpMarkDown": "Time to stop the metric collection. Provide timestamp in yyyy-mm-ddThh:mm:ssZ format (UTC time)"
+        },
+        {
+            "name": "interval",
+            "type": "string",
+            "label": "Interval",
+            "defaultValue": "2m",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "helpMarkDown": "single canary analysis task’s lifetime is composed of multiple loops of metrics collection and evaluation – each happening over a time window as specified under the interval input  (provide unit of time (s, m, h))"
+        },
+        {
+            "name": "step",
+            "type": "string",
+            "label": "Step",
+            "defaultValue": "1s",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "helpMarkDown": ""
+        },
+        {
+            "name": "preprocessor",
+            "type": "pickList",
+            "label": "Preprocessor",
+            "defaultValue": "ksigma",
+            "required": true,
+            "options": {
+                "ksigma": "ksigma",
+                "iqr": "iqr"
+            },
+            "groupName": "preprocessingInputs",
+            "helpMarkDown": ""
+        },
+        {
+            "name": "nanStrategy",
+            "type": "multiLine",
+            "label": "NaN strategy",
+            "required": false,
+            "properties": {
+                "resizable": "true",
+                "rows": "4"
+            },
+            "groupName": "preprocessingInputs",
+            "helpMarkDown": " Multiline input where each line contains one of the two values – remove/replaceWithZeroes. This input specifies how NaN values are to be handled for each metric"
```

Comment:
Extra space in the beginning. Shouldn't this have a default?

Reply:
Checked that it defaults to REMOVE in aa code.
Update help stating the default value.